drivers/lte_link_control/lte_lc.c

```diff
@@ -130,6 +137,11 @@ static int w_lte_lc_init_and_connect(struct device *unused)
 	}
 	LOG_INF("PDP Context: %s", cgdcont);
 #endif
+	err = at_cmd(at_socket_fd, network_mode, AT_CMD_SIZE(network_mode));
+	if (err) {
+		close(at_socket_fd);
+		return err;
```

Comment:
Perhaps a `LOG_ERR()` to give some immediate feedback to the user where things failed.

Reply:
`LOG_ERR()` is in the `at_cmd()` function